### PR TITLE
docs: Add MongoDB page

### PR DESCRIPTION
**Pull request by josephglanville**
Document usage for the MongoDB appliance.

### CODE REVIEW

File: docs/content/mongodb.html.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+$ mongodump -o tmp # you will need to use --host and auth flags as appropriate
+$ cd tmp
+$ tar -cf ../mydb.dump
+```
+
+## Safety
+
+The MongoDB appliance uses a ReplicaSet for replication.
+As such all operations are replicated asyncronously.
+To garuantee durability of writes the [majority write concern](https://docs.mongodb.com/manual/reference/write-concern/) should be used.
```
Comment (titanous) [inline docs/content/mongodb.html.md]: guarantee

**Comment by josephglanville** [comment]
Comments addressed and fleshed out the reasons why the additional client safety options are required.

File: docs/content/mongodb.html.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+## Safety
+
+The MongoDB appliance uses a ReplicaSet for replication.
+As such all operations are replicated asyncronously.
+To guarantee durability of writes the [majority write concern](https://docs.mongodb.com/manual/reference/write-concern/) should be used.
+This will ensure writes are replicated before the write is acknowledged by the client.
+If read consistency/isolation is required then the [majority read concern](https://docs.mongodb.com/manual/reference/read-concern/) must also be utilised.
+This setting ensures writes can only be read if they have been commited to the other cluster members, it is otherwise possible to return uncommited data and potentially stale reads.
```
Comment (titanous) [inline docs/content/mongodb.html.md]: uncommitted

Comment (titanous) [inline docs/content/mongodb.html.md]: committed

**Comment by titanous** [comment]
LGTM pending comments

**Comment by josephglanville** [comment]
@titanous ready for final review.

**Comment by titanous** [comment]
LGTM

**Pull request merged**